Checkpoint-Counting-Cards.md

```diff
@@ -0,0 +1,111 @@
+
+| Value  | Cards                  |
+| ------ |:----------------------:|
+| +1     | 2, 3, 4, 5, 6          |
+| 0      | 7, 8, 9                |
+| -1     | 10, 'J', 'Q', 'K','A'  |
+
+***Instructions***
+
+You will write a card counting function. It will receive a **card** parameter and increment or decrement the global **count** variable according to the card's value (see table). The function will then return the current count and the string **"Bet"** if the count is positive, or **"Hold"** if the count is zero or negative.
+
+**Example Output:**
+
+- -3 Hold
+- 5 Bet
+
+Remember to use [ Read-Search-Ask](http://github.com/FreeCodeCamp/freecodecamp/wiki/How-to-get-help-when-you-get-stuck) if you get stuck. Try to pair program. Write your own code.
+
+## Useful Links
+- [Waypoint: Selecting from many options with Switch Statements](http://www.freecodecamp.com/challenges/waypoint-selecting-from-many-options-with-switch-statements)
+- [Waypoint: Chaining If Else Statements](http://www.freecodecamp.com/challenges/waypoint-chaining-if-else-statements)
+- [Waypoint: Increment a Number with Javascript](http://www.freecodecamp.com/challenges/waypoint-increment-a-number-with-javascript)
+
+## Problem Explanation:
+- Change the code below `// Only change code below this line` and up to `// Only change code above this line`
+- Take note that you are editing the inside of the `cc` function
+- Use what you've learned to check the value of each `card` parameter passed into the function
+- Keep a running count of that number
+- If the final count is 1 or greater, return **# Hold**
+- If the final count is 0 or less, return **# Bet**
+
+## Hint: 1
+- Use a `switch` (or `else if`) statement to count the value of each card.
+
+## Hint: 2
+- Add/subtract the value of each card to variable `count`. If the card is worth 0, don't do anything!
+
+## Hint: 3
+- After you've counted the cards, use an `if` statement to check the value of `count`. Also, make sure your return has a space between the number and the string.
+
+## Spoiler Alert!
+[![687474703a2f2f7777772e796f75726472756d2e636f6d2f796f75726472756d2f696d616765732f323030372f31302f31302f7265645f7761726e696e675f7369676e5f322e676966.gif](https://files.gitter.im/FreeCodeCamp/Wiki/nlOm/thumb/687474703a2f2f7777772e796f75726472756d2e636f6d2f796f75726472756d2f696d616765732f323030372f31302f31302f7265645f7761726e696e675f7369676e5f322e676966.gif)](https://files.gitter.im/FreeCodeCamp/Wiki/nlOm/687474703a2f2f7777772e796f75726472756d2e636f6d2f796f75726472756d2f696d616765732f323030372f31302f31302f7265645f7761726e696e675f7369676e5f322e676966.gif)
+
+**Solution ahead!**
+
+## Code Solution:
+
+```
+function cc(card) {
+  // Only change code below this line
+  switch(card){
+    case 2:
+    case 3:
+    case 4:
+    case 5:
+    case 6:
+      count++;
+      break;
+    case 7:
+    case 8:
+    case 9:
+      break;
```

Comment:
You can technically omit these lines entirely (66-69) because they serve no function.  If not match is found in a switch statement then nothing happens.